test/runtests.jl

```diff
@@ -90,26 +99,32 @@ end
         # clean out any previous coverage files. Don't use clean_folder because we
         # need to preserve the pre-baked coverage file Coverage.jl.cov
         clean_file(srcname)
-        cmdstr = "include(\"$(escape_string(srcname))\"); using Test; @test f2(2) == 4"
+        cmdstr = "include($(repr(srcname))); using Test; @test f2(2) == 4"
         run(`$(Base.julia_cmd()) --startup-file=no --code-coverage=user -e $cmdstr`)
         r = process_file(srcname, datadir)
 
-        target = Union{Int64,Nothing}[nothing, 2, nothing, 0, nothing, 0, nothing, nothing, nothing, nothing, 0, nothing, nothing, nothing, nothing, nothing, 0, nothing, nothing, 0, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing]
+        target = Coverage.CovCount[nothing, 2, nothing, 0, nothing, 0, nothing, nothing, nothing, nothing, 0, nothing, nothing, nothing, nothing, nothing, 0, nothing, nothing, 0, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing, nothing]
+        target_disabled = map(x -> (x !== nothing && x > 0) ? x : nothing, target)
         @test r.coverage == target
 
-        covtarget = (sum(x->x != nothing && x > 0, target), sum(x->x != nothing, target))
+        covtarget = (sum(x->x !== nothing && x > 0, target), sum(x->x != nothing, target))
```

Comment:
Is there a reason to change the first != to !== but not the second?
```suggestion
        covtarget = (sum(x->x !== nothing && x > 0, target), sum(x->x !== nothing, target))
```

Reply:
No reason, just didn't see it. That's why code review is good :)